app/app.go

```diff
@@ -393,6 +394,8 @@ func New(
 		signal.NewAppModule(app.SignalKeeper),
 		minfee.NewAppModule(encodingConfig.Codec, app.ParamsKeeper),
 		packetforward.NewAppModule(app.PacketForwardKeeper, app.GetSubspace(packetforwardtypes.ModuleName)),
+		// ensure the light client module types are registered.
+		ibctm.NewAppModule(),
```

Comment:
path creation was failing in tests with a tx decode error on the celestia app side.